src/components/Navbar/styles.scss

```diff
@@ -0,0 +1,112 @@
+.nav {
+    width: 100%;
+    height: 88px;
+    margin: 0;
+    padding: 0;
```

Comment:
It depends on the situation but I think we shouldn't use fixed heights. Font size and vertical spacing should determine the height.

Reply:
I will try this, thank you